### PR TITLE
Upgrade GUI

### PR DESCRIPTION
**Pull request by PateShin**

### CODE REVIEW

File: src/main/java/seedu/address/ui/TransactionCard.java [hunk 1]
```diff
@@ -37,9 +41,27 @@ public class TransactionCard extends UiPart<Region> {
...
+        String employeeIdLabel = "Employee ID: ";
+        String employeeIdText = Integer.toString(transaction.getEmployeeId().value);
+        employeeId.setText(employeeIdLabel + employeeIdText);
+
+        String descriptionLabel = "Description: ";
+        String descriptionText = transaction.getDescription().value;
+        description.setText(descriptionLabel + descriptionText);
+
+
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm");
```
Comment (DhiraPT) [inline src/main/java/seedu/address/ui/TransactionCard.java:62]: Perhaps can obtain the DATETIME_FORMAT from DateTime class?

File: src/main/java/seedu/address/ui/TransactionCard.java [hunk 2]
```diff
@@ -37,9 +41,27 @@ public class TransactionCard extends UiPart<Region> {
...
+        String dateLabel = "DateTime: ";
+        String dateText = transaction.getDateTime().value.format(formatter);
```
Comment (DhiraPT) [inline src/main/java/seedu/address/ui/TransactionCard.java:64]: I think you can just use the `toString()` method inside DateTime class here.

**Review by DhiraPT — CHANGES_REQUESTED** [review]
Good job on modifying the GUI.

File: src/main/java/seedu/address/ui/TransactionCard.java (hunk 1 above)
Comment (PateShin) [reply src/main/java/seedu/address/ui/TransactionCard.java]: I have removed the `formatter`, and chosen to use `getDateTime` directly from the `transaction`.

**Review by DhiraPT — COMMENTED** [review]
Well done!

**Review by DhiraPT — APPROVED** [review]

**Review by ondretann — COMMENTED** [review]
LGTM!

**Pull request merged**